inductiva/storage/storage.py

```diff
@@ -17,11 +17,13 @@ def get_space_used():
         raise api_exception
 
 
-def listdir(max_results: int = 3,
+def listdir(path="/",
+            max_results: int = 10,
             order_by: Literal["size", "creation_time"] = "size",
             sort_order: Literal["asc", "desc"] = "desc"):
     """List and display the contents of the user's storage bucket.
     Args:
+        path (str) : directory inside storage to list from. default is root.
```

Comment:
Default*

Reply:
Directory*

Reply:
(str):* (no space in between)

Reply:
Maybe "Storage directory to list"

Reply:
You won't need this redudancy, since you already mention above that you will list from this storage path